docs/digger/installation/digger-prereqs.asciidoc

```diff
@@ -1,22 +1,45 @@
 == Prerequisites
 
-A Digger server can be installed using the https://github.com/aerogear/digger-installer[Digger Installer tool].
+The main components (jenkins, android-sdk, nagios) of the digger project are deployed to an OpenShift Container Platform Installation. With these components installed digger has the capability to build Android Applications. The user has the option to configure (via the digger installer) a mac server to allow for iOS Application builds. This is an optional step and the Mac Server pre-requisites section below can be ignore if a Mac Server is not part of the installation.
```

Comment:
"to an OpenShift Container Platform Installation" ---> "to an OpenShift Container Platform cluster" or "OpenShift Container Platform"

"With these components installed digger" ---> "Withe these components installed, digger"

"mac" --> "macOS" for consistency

"ignore"  ---> "ignored"